GTMAdminOps/about.rst

```diff
@@ -8,13 +8,13 @@
     About YottaDB
 
 ==========================
- About YottaDB
+ About YottaDB/GT.M
 ==========================
 
 .. contents::
    :depth: 2
 
-YottaDB is a high-end performance database application development and deployment platforms from YottaDB LLC. YottaDB provides an M language subsystem, a database engine, and a complete set of utilities for administration and operation. One can install and manage YottaDB using the utilities described in this manual and standard operating system tools. This chapter provides an overview of the YottaDB system environment and utilities. 
+YottaDB/GT.M are high-end performance database application development and deployment platforms from YottaDB LLC and FIS. YottaDB/GT.M provides an M language subsystem, a database engine, and a complete set of utilities for administration and operation. One can install and manage YottaDB/GT.M using the utilities described in this manual and standard operating system tools. This chapter provides an overview of the YottaDB/GT.M system environment and utilities. 
```

Comment:
The singular, as in "YottaDB/GT.M is a high-end" sounds better. Otherwise, we should say "YottaDB and GT.M are high-end" which would get wordy. For now, since it's one manual, let's pretend it's in the singular, and as we edit each section, we can drop the GT.M and retain the singular (it will make for less future editing too).